### PR TITLE
Update bidspm naming and add possibility to create gin repo from terminal

### PR DESCRIPTION
**Pull request by marcobarilari**
The sibling setting via existing URL can be deprecated since it could be more handy to just create anything in output from scratch since I doubt about any use case in which one create a yoda repo and has already outputs repos somewhere (but I might be wrong)

### CODE REVIEW

File: dataladConfig.sh [hunk 1]
```diff
@@ -11,15 +11,19 @@ URL_RAW=''
...
+# for DEMO use: git@gin.g-node.org:/cpp_brewery/CPP_visMotion-bidspm-stats.git
 URL_DER_STATS=''
 
+# GIN base name
+# to create siblings automatically for each new repo (i.e.  analysis_visualMotionLoc)
+GIN_BASENAME=''
+
 # set to "true" instead of "false" if you want to use the bleeding edge version of CPP SPM
-USE_CPP_SPM_DEV=false
+USE_BIDSPM_DEV=false
```
Comment (Remi-Gau) [inline dataladConfig.sh:26]: I would probably nuke this and its use in the code as well.

Dev is most often lagging being main these days.

File: dataladSetUp.sh [hunk 2]
```diff
@@ -43,17 +53,22 @@ if [ ! -z "${URL_DER_STATS}" ]; then
...
+    cd "${stats_dir}"
+    datalad create-sibling-gin -d . -s origin --access-protocol ssh --private  cpp_brewery/"${GIN_BASENAME}"-derivatives-bidspm-stats
     cd "${root_dir}"
+    datalad subdatasets --set-property url https://gin.g-node.org/cpp_brewery/"${GIN_BASENAME}"-derivatives-bidspm-stats bidspm-stats
 fi
 
-if [ "${USE_CPP_SPM_DEV}" = true ]; then
-    cd code/lib/CPP_SPM
+if [ "${USE_BIDSPM_DEV}" = true ]; then
+    cd code/lib/bidspm
```
Comment (Remi-Gau) [inline dataladSetUp.sh:66]: See comment above

**Review by Remi-Gau — APPROVED** [review]
Looks good to me.
May want to remove the option to use the dev branch of bidspm. Up to you.

**Comment by marcobarilari** [comment]
| May want to remove the option to use the dev branch of bidspm. Up to you.

good idea!

I am leaning to leave only the option to create the the repo both locally and remotely and delete the part that add the the siblings manually. I really do not think about a use case (at least in my experience) in which I would prefer to add the siblings manually. In the case a repo online already exists before, it is smarter to use `datalad install` instead creating an empty repo locally and then link local and remote.

Thanks!

**Pull request merged**